Themes/default/languages/index.english.php

```diff
@@ -152,6 +152,7 @@
 $txt['unwatch_board'] = 'Stop watching Board';
 $txt['watch_topic'] = 'Watch this Topic';
 $txt['unwatch_topic'] = 'Stop watching Topic';
+$txt['watching_topic'] = 'You are watching the topic';
```

Comment:
"Topic you are watching" would be more consistent with the phrasing used in the rest of the legend.

Reply:
alright 12ddba9